### PR TITLE
Fix CVE-2021-23346 vulnerability

### PR DESCRIPTION
**Pull request by MAOKG**
* update react-i18next to 11.8.13 to fix [CVE-2021-23346](https://github.com/i18next/react-i18next/issues/1275)

### CODE REVIEW

File: examples/simple/yarn.lock [hunk 1]
```diff
@@ -2485,15 +2485,8 @@ neo-async@^2.5.0, neo-async@^2.6.1, neo-async@^2.6.2:
...
-  dependencies:
-    "@types/hoist-non-react-statics" "^3.3.1"
-    "@types/i18next-fs-backend" "^1.0.0"
-    core-js "^3"
-    hoist-non-react-statics "^3.2.0"
-    i18next "^20.1.0"
-    i18next-fs-backend "^1.0.7"
-    react-i18next "^11.8.8"
+  version "0.0.0"
+  uid ""
```
Comment (isaachinman) [inline examples/simple/yarn.lock:2489]: These changes should be removed.

**Review by isaachinman — CHANGES_REQUESTED** [review]

File: examples/simple/yarn.lock (hunk 1 above)
Comment (MAOKG) [reply examples/simple/yarn.lock:2489]: Oh, it's added by yarn when running `yarn test`. Should I remove all the changes on `examples/simple/yarn.lock`? 

My yarn version is 1.22.10

Comment (isaachinman) [reply examples/simple/yarn.lock:2489]: You can leave the other changes, but the `next-i18next` entry should not be changed.

Comment (MAOKG) [reply examples/simple/yarn.lock]: Thanks, @isaachinman, I just pushed the update

**Comment by MAOKG** [comment]
@isaachinman Can you have another look when you have time? Hopefully, this change will get released soon, [CVE-2021-23346](https://github.com/i18next/react-i18next/issues/1275) is blocking us from consuming `next-i18next`. Thanks in advance!

**Review by isaachinman — APPROVED** [review]

**Pull request merged**